modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.wiki.constants;
+
+/**
+ * @author Janis Zhang
+ */
+public class WikiImporterKeys {
+
+	public static final String FORMAT_CREOLE = "creole";
+
+	public static final String OPTIONS_FRONT_PAGE = "OPTIONS_FRONT_PAGE";
+
+	public static final String OPTIONS_IMPORT_LATEST_VERSION =
+		"OPTIONS_IMPORT_LATEST_VERSION";
+
+	public static final String OPTIONS_STRICT_IMPORT_MODE =
+		"OPTIONS_STRICT_IMPORT_MODE";
+
+	public static final String SHARED_IMAGES_CONTENT = "See attachments";
+
+	public static final String SHARED_IMAGES_TITLE = "SharedImages";
```

Comment:
@JanisZhang This one is used by several Java classes, which should be in a class named `*Constants`.

Reply:
got it